<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup2</class>
 <widget class="QDialog" name="Signup2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>711</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>signup</string>
   </property>
   <widget class="QPushButton" name="signUp_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Password : </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="student_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Student</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="instractur_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instractur</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="admin_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>adminstrator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>User Id : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userId_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
